prime-router/metadata/schemas/FL/fl-covid-19.schema

```diff
@@ -250,7 +266,6 @@ elements:
 
     # column AY
     - name: employed_in_healthcare
-      type: CODE
       csvFields: [{ name: Employment in Health Care }]
       altValues:
           - display: Yes
```

Comment:
I suppose we should be grateful there are so may ways to say 'Yes'. ;) 
Per my previous comment, let's create a `TABLE` with all of them.